l3build-file-functions.lua

```diff
@@ -302,10 +301,10 @@ function tree(path, glob)
   end
   local dirs = {["."] = cropdots(path)}
   for pattern, criterion in gmatch(cropdots(glob), "([^/]+)(/?)") do
-    local criterion = criterion == "/" and is_dir or always_true
-    local function fill(path, dir, table)
+    criterion = criterion == "/" and is_dir or always_true
+    local function fill(path_a, dir, table)
```

Comment:
`criterion` is already local
`path` is already used

Reply:
About `path`: I probably don't surprise anyone when I say that I don't like non-descriptive names like `path_a`. `current_path`? `subpath`?

Also: Is there a reason why you didn't change the name in `cropdots` ~10 lines higher? Since that's a one-liner I don't think that it's an issue, but I had the impression that you suggested it pretty consistently everywhere else.